cmd/lncli/main.go

```diff
@@ -342,38 +342,44 @@ func main() {
 	app.Usage = "control plane for your Lightning Network Daemon (lnd)"
 	app.Flags = []cli.Flag{
 		cli.StringFlag{
-			Name:  "rpcserver",
-			Value: defaultRPCHostPort,
-			Usage: "The host:port of LN daemon.",
+			Name:   "rpcserver",
+			Value:  defaultRPCHostPort,
+			Usage:  "The host:port of LN daemon.",
+			EnvVar: "LNCLI_RPCSERVER",
```

Comment:
Cool, I wasn't even aware that the CLI library supported this feature. Very cool!

Reply:
It is, so it is "free" to add, well tested and self-documenting :)